python/paddle/fluid/backward.py

```diff
@@ -933,6 +933,31 @@ def _append_backward_ops_(block,
                 cb(block=target_block, context=grad_to_var)
 
 
+def _is_grad_var_(var):
+    return core.grad_var_suffix() in var
+
+
+# Find the op whose holds the sub_block as its "sub_block" attr
```

Comment:
NIT: "Find the op whose holds" -> "Find the op who holds"

Reply:
Done.